dags/bqetl_internet_outages.py

```diff
@@ -32,6 +32,18 @@
         dag=dag,
     )
 
+    wait_for_copy_deduplicate_main_ping = ExternalTaskSensor(
+        task_id="wait_for_copy_deduplicate_main_ping",
+        external_dag_id="copy_deduplicate",
+        external_task_id="copy_deduplicate_main_ping",
+        execution_delta=datetime.timedelta(seconds=7200),
+        check_existence=True,
+        mode="reschedule",
+    )
+
+    internet_outages__global_outages__v1.set_upstream(
+        wait_for_copy_deduplicate_main_ping
+    )
```

Comment:
Ideally, we'd be able to avoid defining this extra sensor since we know that this query depends on clients_daily, and that clients_daily already depends on copy_dedupe_main_ping. Not necessary here, but would be nice to have.